src/api/SFA.DAS.Provider.Events.Api.IntegrationTests/Setup/DatabaseSetup.cs

```diff
@@ -71,7 +95,7 @@ public DatabaseSetup(PopulateTables populate)
 
         private async Task ReadSubmissionEvents()
         {
-            const string submissionEventsSql = "SELECT * FROM [Submissions].[SubmissionEvents]";
+            const string submissionEventsSql = "SELECT * FROM [Submissions].[SubmissionEvents] WHERE Id > 3";
```

Comment:
Not sure using the id to filter is a good idea.  Could we use uln = null instead?